src/pages/Heeyun/Detail/Detail.js

```diff
@@ -0,0 +1,232 @@
+import React from 'react'
+import './Detail.scss';
+import TopNav from '/Users/heeyun/Desktop/wecode/fullstack-3-React-Webucks-3/src/components/Nav/TopNav';
+import ImgDetail from '../../../components/imgList/ImgDetail';
+import { useState, useEffect } from 'react';
+
+const Detail = () => {
+
+    const [imgData, setImgData] = useState([]);
+
+    useEffect(() => {
+        fetch('http://localhost:3000/data/imgDetailData.json', {
+        method: 'GET' 
+        })              
+        .then(res => res.json())
+        .then(data => {
+            setImgData(data);
+        });
+    }, [])
+
+    const [heartColor, setHeartColor] = useState('black')
+    function clickHeart(){
+        if(heartColor==='red'){
+            return setHeartColor('black')
+        }
+        if(heartColor==='black'){return setHeartColor('red')}
```

Comment:
이 부분은 else로 수정하시면 좋을 것 같아요!
```suggestion
        else {
             return setHeartColor('red')
        }
```